source/frontend/src/components/common/ModalContainer.tsx

```diff
@@ -5,11 +5,7 @@ import { ModalContext } from '@/contexts/modalContext';
 
 import GenericModal from './GenericModal';
 
-interface IModalContainerProps {}
-
-export const ModalContainer: React.FunctionComponent<
-  React.PropsWithChildren<IModalContainerProps>
-> = props => {
+export const ModalContainer: React.FunctionComponent<React.PropsWithChildren<unknown>> = () => {
```

Comment:
I am guessing there is a non prop version?

Reply:
this is just cleaning up unused variables - in this case props is never used.